kubernetes/compose/v1beta1/doc.go

```diff
@@ -1,11 +1,10 @@
-// Package v1beta1 holds the v1beta1 versions of our stack structures.
```

Comment:
Hm, why was the Package description move further down? The first line is what will be documented as package description